devices/sonoff/snzb-03p_motion_sensor.json

```diff
@@ -0,0 +1,192 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "eWeLink",
+  "modelid": "SNZB-03P",
+  "vendor": "SONOFF",
+  "product": "SNZB-03P",
+  "sleeper": false,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$TYPE_PRESENCE_SENSOR",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0406"
+      ],
+      "fingerprint": {
+        "profile": "0x0104",
+        "device": "0x0107",
+        "endpoint": "0x01",
+        "in": [
+          "0x0000",
+          "0x0001",
+          "0x0406",
+          "0x0500",
```

Comment:
Just wondering if the device actually uses the IAS Zone cluster or if it has any effect. I'd conclude the device doesn't require any enrollment to work?

Reply:
I think no
And there is not config/enrollemetn and config/pending.

Removed.